src/index.html

```diff
@@ -22,6 +22,323 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="container">
+        <a
+          href="#home"
+          class="logo"
+        >
+          <img
+            class="logo"
+            src="images/logo.png"
+            alt="logo"
+          />
+        </a>
+
+        <nav class="nav">
+          <ul class="nav_list">
```

Comment:
If you using BEM, after block name, in this case it is `nav` you need to write double underscore (`__`)